packages/babel-generator/src/node/parentheses.js

```diff
@@ -59,8 +59,12 @@ export function UpdateExpression(node: Object, parent: Object): boolean {
   return (
     // (foo++).test(), (foo++)[0]
     t.isMemberExpression(parent, { object: node }) ||
+    // (foo++)?.test(), (foo++)?.[0]
+    t.isOptionalMemberExpression(parent, { object: node }) ||
```

Comment:
Perhaps we should be adding an alias for these, so we don't need to duplicate everywhere.

Reply:
Yeah, I agree.

Reply:
It's really difficult to come up with a proper name other than `CallExpression` itself. 😄

Reply:
`MemberExpressionish` 😛

Reply:
Yah, I was thinking `isMemberish` and `isCallish`.